packages/layer-composer/src/generators/types.ts

```diff
@@ -165,6 +170,7 @@ export interface HeatmapGeneratorConfig extends GeneratorConfig {
   colorRamp?: ColorRampsIds
   serverSideFilter?: string
   updateColorRampOnTimeChange?: boolean
+  legend?: HeatmapGeneratorLegend
```

Comment:
I'm wondering about this... The generator is merely passing that config down to the style, not doing anything with it.
Nothing to fix now but what is your opinion about:
- not passing this config to the generator, but provide a dictionary `generator id/type: legendConfig` to `<MapLegend>`
- or, passing this config to the generator and have the generator produce a semi-rendered legend, like it would already have the legend title generated, and `<MapLegend>` would be dumber?